preparing-for-ada/README.md

```diff
@@ -1,11 +1,29 @@
 # Ada Developers Academy JumpStart Curriculum
 
+## The Ada Application Process
+
+You can read more about the Ada application process [on our website](https://adadevelopersacademy.org/process/).  With the online application you will submit a resume and answer some questions.  In Phase 2, you will be confronted with a coding challenge in [Python](https://www.python.org/) to solve and an interview to explain their solution and examine ways to extend it.  Applicants who proceed to Phase 3 will participate in a video interview.
```

Comment:
I think it makes sense to link them to interview process on our website rather than repeating those instructions here. Should we give a little mention about what Phase 3 will involve (technical challenge and behavioral questions?).